test/langtools/jdk/javadoc/doclet/testUnicode/TestUnicode.java

```diff
@@ -42,27 +41,90 @@
 
     public static void main(String... args) throws Exception {
         TestUnicode tester = new TestUnicode();
-        tester.runTests();
+        tester.runTests(m -> new Object[] { Path.of(m.getName())});
     }
 
     ToolBox tb = new ToolBox();
 
     @Test
-    public void test() throws Exception {
+    public void testUnicode(Path base) throws Exception {
         char ellipsis = '\u2026';
-        Path src = Files.createDirectories(Paths.get("src"));
+        Path src = Files.createDirectories(base.resolve("src"));
         tb.writeJavaFiles(src,
                 "/** Hel" + ellipsis + "lo {@code World(" + ellipsis + ")}. */\n"
                 + "public class Code { }\n");
 
-        javadoc("-d", "out",
+        javadoc("-d", base.resolve("out").toString(),
                 "-encoding", "utf-8",
                 src.resolve("Code.java").toString());
         checkExit(Exit.OK);
 
         checkOutput("Code.html", true,
-                "<div class=\"block\">Hel" + ellipsis + "lo <code>World(" + ellipsis + ")</code>.</div>");
+                """
+                        """);
```

Comment:
Is don't think this is supposed to be an empty string, is it?

Reply:
Oops, will investigate what happened in my repo.

Reply:
This could've been detected during testing, if we had https://github.com/openjdk/jdk/pull/4811.